CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file based on the
 
 ### Backward Compatibility Fixes
 - Updated Elastica\Test\Suggest\CompletionTest now payload and output are removed
+- Updated Elastica\Test\TypeTest::testGetDocumentWithFieldsSelection The stored_fields parameter will only return stored fields — it will no longer extract values from the _source
```

Comment:
I think when we just update tests and don't to any chances to the actual code, we don't need the changelog normally. Going to merge it anyways ;-)